2.7.0/todo/result_distributed.md

```diff
@@ -30,7 +30,10 @@ The categories below are as follows:
 ### deprecation
 ### new features
 - Context parallel
-  - [place holder for CP description] ([#132820](https://github.com/pytorch/pytorch/pull/132820), [#142093](https://github.com/pytorch/pytorch/pull/142093), [#148377](https://github.com/pytorch/pytorch/pull/148377))
+  - We provided a [Context Parallel API](https://github.com/pytorch/pytorch/pull/131351) for users to parallelize `torch.nn.functional.scaled_dot_product_attention` over the sequence dimension. We implemented
+  [Ring Attention](https://github.com/pytorch/pytorch/pull/131351) and an [AllGather-based approach](https://github.com/pytorch/pytorch/pull/132820) where the all-gather happens after the local SDPA,
```

Comment:
> where the all-gather happens after the local SDPA,

where the all-gather is issued before the first local SDPA and the subsequence local SDPAs will have to wait for the all-gather.